dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/CollectionConverter.java

```diff
@@ -66,6 +66,11 @@ public CollectionRest fromModel(org.dspace.content.Collection obj) {
 
         col.setDefaultAccessConditions(getDefaultBitstreamPoliciesForCollection(obj.getID()));
 
+        try {
+            col.setOwningCommunity(obj.getCommunities().get(0).getID().toString());
```

Comment:
I'm worried this is a major assumption.  Collections don't have an "owning Community" (in the same way that an Item does).  We shouldn't assume the first Community listed is the owner, as the relationship between a Collection and its parent Community(ies) is unordered.  See also my comments on the REST Contract PR here: https://github.com/DSpace/Rest7Contract/pull/33#discussion_r239602991